mats/unix.ms

```diff
@@ -72,8 +72,8 @@
   (mat system
     (eqv? (with-output-to-file "testfile.ss" void '(replace)) (void))
     (begin
-      (system (format "~:[~;/pkg~]/bin/rm testfile.ss" (embedded?)))
-      (system (format "~:[~;/pkg~]/bin/echo hello > testfile.ss" (embedded?)))
+      (system (format "~:[~;/pkg~]rm testfile.ss" (embedded?)))
+      (system (format "~:[~;/pkg~]echo hello > testfile.ss" (embedded?)))
```

Comment:
This seems not quite right to me.  When `(embedded?)` is `#t` it is going to generate `/pkgrm` an `/pkgecho`, I think the intention was to get either `/pkg/bin/rm` or `/pkg/bin/echo`, if the intention is to always use `rm` or `echo` we could remove the `format` and just have `"rm tetfile.ss"` an `"echo hello > test file.ss"`, though we should probably consider adding `-f` for `rm`.

Reply:
I don't think I understand what's going on here. The only definition of `embedded?` I can find is: https://github.com/cisco/ChezScheme/blob/17fd84de28234e3a3e1dc29223a99ab466fc6b87/mats/mat.ss#L506-L507
for which, as with these lines in `mats/unix.ms`, the git blame points to “initial upload of open-source release”. I'm not thinking of a system or context when `/pkg` would usually exist: I guessed it might refer to the support for building Mac `.pkg` packages, but I'm not seeing anything likely there.

(For posterity, [apparently Common Lisp specifies](http://www.lispworks.com/documentation/HyperSpec/Body/22_cgb.htm) that the `format` control string “`~:[`*alternative*`~;`*consequent*`~]` selects the *alternative* control string if *arg* is *false*, and selects the *consequent* control string otherwise.”)

Reply:
I don't know that `embedded?` is actually being used right now, but I've not spent enough time going through the code to see if this is something extraneous.  Either way, the change from the Racket leaves the possibility of generating the incorrect `/pkgrm` and `/pkgecho`.  I'm just suggesting that we should avoid the whole issue here and simply remove the formatting, which only exist to add `/pkg`.
```suggestion
      (system "rm -f testfile.ss")
      (system "echo hello > testfile.ss")
```

Reply:
Yes, if that seems good enough for you, it's certainly the simplest.